tests/advect_mesh_vertically/statistics

```diff
@@ -15,24 +15,24 @@
+ 6 2.211996178360e+03 4.169311840033e+02 136 1482  653  653 0  9  14  16  80                                                    "" -1.74367151e+03 6.65722355e+02 2.56801962e-01 7.15037235e-01 
+ 7 2.649671268530e+03 4.376750901699e+02 136 1482  653  653 0  9  15  17  83                                                    "" -2.01885582e+03 7.40213440e+02 2.38432655e-01 6.76166151e-01 
+ 8 3.112578719724e+03 4.629074511946e+02 136 1482  653  653 0  9  14  16  76                                                    "" -2.28920278e+03 8.05083940e+02 2.21910521e-01 6.39259838e-01 
+ 9 3.602205867398e+03 4.896271476736e+02 136 1482  653  653 0  9  13  15  73                                                    "" -2.55478374e+03 8.60548467e+02 2.07085325e-01 6.04371619e-01 
+10 4.120136388632e+03 5.179305212345e+02 136 1482  653  653 0  9  11  13  62                                                    "" -2.81516434e+03 9.07046969e+02 1.94081345e-01 5.72112968e-01 
+11 4.667367515170e+03 5.472311265379e+02 190 2085  921  921 0  9  15  17  85                                                    "" -3.07024403e+03 9.45141408e+02 1.83955069e-01 5.46774470e-01 
+12 5.238953994056e+03 5.715864788856e+02 190 2085  921  921 0  8  15  17  85                                                    "" -3.31903241e+03 9.75741071e+02 1.74061970e-01 5.18832055e-01 
+13 5.841353373919e+03 6.023993798632e+02 190 2085  921  921 0  8  13  15  74                                                    "" -3.56151073e+03 9.99284226e+02 1.65493151e-01 4.93370296e-01 
+14 6.474861718796e+03 6.335083448770e+02 190 2085  921  921 0  8  13  15  75                                                    "" -3.79718369e+03 1.01639162e+03 1.58125916e-01 4.70308248e-01 
+15 7.139479446940e+03 6.646177281436e+02 190 2085  921  921 0  8  14  16  80                                                    "" -4.02583721e+03 1.02786068e+03 1.51868566e-01 4.49988243e-01 
+16 7.834139939604e+03 6.946604926640e+02 232 2487 1099 1099 0  8 204 297 945                                                    "" -4.24736635e+03 1.03454127e+03 1.46649778e-01 4.32412417e-01 
+17 8.557022025416e+03 7.228820858124e+02 232 2487 1099 1099 0  8 203 289 951                                                    "" -4.46158686e+03 1.03751816e+03 1.42299927e-01 4.16954513e-01 
+18 9.306644450899e+03 7.496224254831e+02 232 2487 1099 1099 0  8 202 263 937                                                    "" -4.66800971e+03 1.03879161e+03 1.38655362e-01 4.03374485e-01 
+19 1.008145155745e+04 7.748071065528e+02 232 2487 1099 1099 0  8 204 312 967 output-advect_mesh_vertically/solution/solution-00001 -4.86664453e+03 1.03854521e+03 1.35606730e-01 3.91449902e-01 
+20 1.087985689896e+04 7.984053415080e+02 232 2487 1099 1099 0  8 201 245 933                                                    "" -5.05764216e+03 1.03605211e+03 1.33084178e-01 3.81287482e-01 
```

Comment:
This is a nice improvement in number of iterations, BTW. I don't know how that translates into CPU time, but because the operations in the actual solver are so cheap compared to applying the preconditioner, I suspect that this also translates into CPU time.